app/views/social_networking/goals/tool.html.erb

```diff
@@ -107,4 +107,6 @@
 <script>
   angular.module('socialNetworking.values').value('participantStudyEndDate', '<%= current_participant.active_membership_end_date %>');
   angular.module('socialNetworking.values').value('currentGoals', <%= goals.to_json.html_safe %>);
+  angular.module('socialNetworking.values').value('noticesEnabled', <%= Rails.configuration.enable_notices %>);
+  angular.module('socialNetworking.values').value('noticeUtility', Notice);
```

Comment:
Are you using `Notice` in other places? Is that why you're injecting it as a global variable?